### PR TITLE
added image and button component

### PR DESCRIPTION
**Pull request by shristabaruah**
--added button component
--added image component
button1
button2
button3
button4
image1
image2
image3

### CODE REVIEW

**Review by sagar455 — APPROVED** [review]
looks good

File: components/image/image.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+
+                </div>
+
+
+
+
+                 
+
```
Comment (AtulPant2704) [inline components/image/image.html:100]: You can remove this extra space.

**Review by AtulPant2704 — APPROVED** [review]
Other than that looks fine.

File: components/image/image.html (hunk 1 above)
Comment (shristabaruah) [reply components/image/image.html:100]: yes ,will do that.

**Pull request merged**